inc/options/common/top_nav_bar.php

```diff
@@ -0,0 +1,122 @@
+	Util_UI::admin_url( 'admin.php?page=w3tc_install' )          => array( 'text' => esc_attr__( 'Install', 'w3-total-cache' ) ),
+	'https://api.w3-edge.com/v1/redirects/faq'                   => array(
+		'text'   => 'FAQ',
+		'target' => '_blank',
+	),
+);
+
+do_action( 'w3tc-dashboard-top-nav-bar' );
+?>
+<div id="w3tc-top-nav-bar">
+	<div id="w3tc-top-nav-bar-content">
+		<a class="logo-link" href="<?php echo esc_url( Util_UI::admin_url( 'admin.php?page=w3tc_dashboard' ) ); ?>" alt="W3 Total Cache">
+			<h2 class="logo">
+				<?php
+				echo wp_kses(
+					sprintf(
+						// translators: 1 opening HTML span tag, 2 opening HTML sup tag, 3 closing HTML sup tag, 4 closing HTML span tag.
+						__(
+							'W3 Total Cache %1$sby W3 EDGE %2$s&reg;%3$s%4$s',
+							'w3-total-cache'
+						),
+						'<span>',
+						'<sup>',
+						'</sup>',
+						'</span>'
+					),
+					array(
+						'span' => array(),
+						'sup'  => array(),
+					)
+				);
+				?>
+			</h2>
+		</a>
+		<div id="w3tc-top-nav-bar-content-links">
+			<a class="w3tc-top-nav-support" href="<?php echo Util_UI::admin_url( 'admin.php?page=w3tc_dashboard' ); ?>" alt="<?php esc_attr_e( 'Dashboard', 'w3-total-cache' ); ?>">
+				<?php esc_html_e( 'Dashboard', 'w3-total-cache' ); ?>
+			</a>
+			<div class="w3tc-top-nav-dropdown">
+				<a class="w3tc-top-nav-settings" href="<?php echo Util_UI::admin_url( 'admin.php?page=w3tc_general' ); ?>" alt="<?php esc_attr_e( 'Settings', 'w3-total-cache' ); ?>">
+					<?php esc_html_e( 'Settings', 'w3-total-cache' ); ?><span class="dashicons dashicons-arrow-down-alt2"></span>
+				</a>
+				<div id="w3tc-top-nav-settings-menu" class="w3tc-top-nav-dropdown-content">
+					<?php
+					foreach ( $settings_menu_array as $url => $data ) {
+						$target = ! empty( $data['target'] ) ? ' target="' . $data['target'] . '"': '';
+						$external = parse_url( $url, PHP_URL_HOST ) !== $_SERVER['HTTP_HOST'] ? '<span class="dashicons dashicons-external"></span>': '';
+						echo '<a href="' . $url . '" alt="' . $data['text'] . '"' . $target . '>' . $data['text'] . $external . '</a>';
+					}
+					?>
+				</div>
+			</div>
+			<div class="w3tc-top-nav-dropdown">
+				<a class="w3tc-top-nav-tools" href="<?php echo Util_UI::admin_url( 'admin.php?page=w3tc_extensions' ); ?>" alt="<?php esc_attr_e( 'Tools', 'w3-total-cache' ); ?>">
+					<?php esc_html_e( 'Tools', 'w3-total-cache' ); ?><span class="dashicons dashicons-arrow-down-alt2"></span>
+				</a>
+				<div id="w3tc-top-nav-tools-menu" class="w3tc-top-nav-dropdown-content">
+					<?php
+					foreach ( $tools_menu_array as $url => $data ) {
+						$target = ! empty( $data['target'] ) ? ' target="' . $data['target'] . '"': '';
+						$external = parse_url( $url, PHP_URL_HOST ) !== $_SERVER['HTTP_HOST'] ? '<span class="dashicons dashicons-external"></span>': '';
+						echo '<a href="' . $url . '" alt="' . $data['text'] . '"' . $target . '>' . $data['text'] . $external . '</a>';
```

Comment:
Escape later rather than earlier.  Escape URLs with `Util_UI::admin_url()`, target with `esc_attr()`, and the text can be translated earlier with `__()` above and then here with `esc_html()`.